drac_config

```diff
@@ -174,6 +175,10 @@ while [ $# -ne 0 ]; do
         "enableipmi")
             sh $libdir/ipmi.sh
             ;;
+        "setwebname")
+            export racadm_binary=$RACADM
```

Comment:
Because `$racadm` that gets exported already comes with the `-r` argument set to the hostname. 
This specific command needs to use the IP address instead.

The comment on the lib/webservername.sh file explains it.

Reply:
I actually don't need to make a new variable here. I can just export the existing `$RACADM`. I pushed an updated commit.